zzk/service/hostregistry.go

```diff
@@ -250,8 +251,21 @@ func (h *HostRegistryListener) Spawn(cancel <-chan interface{}, hostid string) {
 						// soon as possible.
 						select {
 						case <-h.isOnline:
-							count := DeleteHostStates(h.conn, h.poolid, hostid)
+							// Only reschedule services without address
+							// assignments.
+							count := DeleteHostStatesWhen(h.conn, h.poolid, hostid, func(s *State) bool {
+								return s.DesiredState == service.SVCStop || s.AssignedIP == ""
```

Comment:
Will this prevent scheduling services that have a VIP assignment as well as a static?  Is that something we watn?